gbpservice/neutron/nsf/config_agent/firewall.py

```diff
@@ -0,0 +1,154 @@
+import sys
+import ast
+
+from oslo_config import cfg
+from oslo_messaging import target
+from oslo_log import log as logging
+from neutron import manager
+
+from neutron_fwaas.db.firewall import firewall_db
+
+from gbpservice.neutron.nsf.config_agent import RestClientOverUnix as rc
+from gbpservice.neutron.nsf.config_agent import topics
+from neutron.common import rpc as n_rpc
+
+LOG = logging.getLogger(__name__)
+
+Version = 'v1'  # v1/v2/v3#
+
+
+class Firewall(object):
```

Comment:
Added RPCClient class in common.py which is generic to all service types.
class previously was instantiated in rpc_cb.py,with the latest changes it will just call the function name from rpc_cb.py and,RPCClient is instanciated in the function.